conjure-java-core/src/integrationInput/java/com/palantir/product/DoubleAliasExample.java

```diff
@@ -42,6 +42,11 @@ public static DoubleAliasExample of(double value) {
         return new DoubleAliasExample(value);
     }
 
+    @JsonCreator
+    public static DoubleAliasExample of(long value) {
+        return new DoubleAliasExample((double) value);
```

Comment:
I think we probably need to check if the cast would end up imprecise on this

Reply:
I updated the code to only support numbers up to 2^53. That said, I don't think this is quite right. If I pass in a large double, jackson will happily deserialize lossily. I'm not sure why the behavior should be different when the `.` character is omitted.

Reply:
oof, we probably ought to figure out what Jackson settings we need to set to at least be able to log on floating point values that double can't represent

Reply:
How would we detect that?

Reply:
Can we merge this fix for now? Happy to own making the next change